cmd/guard.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/hashicorp/go-version"
+	"github.com/leancodepl/poe2arb/flutter"
+	"github.com/spf13/cobra"
+)
+
+type flutterConfigKey int
+
+const key flutterConfigKey = 1
```

Comment:
Use the same convention as `loggerKey`

Reply:
I changed it, but shouldn't we make `ctxKey` string-based? It will be less error-prone regarding `int` values conflicts.

Reply:
Sorry, I meant here to use the same syntax for defining the constant, i.e. `const key = flutterConfigKey(1)`.

But let me propose another solution for `loggerKey` and `key` of flutter config - `type flutterConfigKey struct{}` and analogously for the logger (https://stackoverflow.com/a/68100270/3158312) now that we have more than one key and they're in different files. Thanks to this we're not using the same type in different files (bigger chance of a mistake by creating a key with the same integer value) and we're not moving all keys to one file just to make it more discoverable.